src/utils/Video.js

```diff
@@ -32,14 +61,17 @@ class Video {
         stream = this.videoElt.captureStream();
       }
       this.video.srcObject = stream;
-      this.video.width = this.size;
-      this.video.height = this.size;
+      if (this.size) {
+        this.video.width = this.size;
+        this.video.height = this.size;
+      }
       this.video.autoplay = true;
       this.video.playsinline = true;
       this.video.muted = true;
       const playPromise = this.video.play();
       if (playPromise !== undefined) {
         playPromise.then(() => {
+          this.videoReady = true;
```

Comment:
Positive affirmation:
* nice one! This is def the right place to update the `this.videoReady` flag. 👍